src/data/faq.json

```diff
@@ -20,14 +20,14 @@
-                            "If the test result is not available in the Corona-Warn-App, the relevant service personnel, test centre or health authority should be contacted to obtain the test result.  <br/>In order to make it easier for you to find the responsible health authority, the RKI has provided a corresponding tool. By entering the postal code or the city of residence, the responsible health authority (including contact details) is immediately displayed: <a href='https://tools.rki.de/PLZTool/en-GB'>Robert Koch Institute Tool: Health authority by postal code or place</a> <br/><a href='../blog/2020-08-25-notes-qr-codes/'>Further information on the QR Code procedure</a>",
+                            "The technical hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to see the phone numbers</a>) will help you with technical questions about the Corona-Warn-App, for instance if you have problems with risk identification. You can reach the technical hotline from Monday to Saturday from 7 a.m. to 10 p.m. (except on German national holidays), and it is free of charge for you within Germany.",
+                            "The TAN hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to see the phone numbers</a>) will help you to enter the TAN if you have a positive test result and if you have not received a TAN or document with a QR code. The TAN hotline is available 24 hours a day, seven days a week. The call is free of charge from within Germany.",
+                            "If the test result is not available in the Corona-Warn-App, the relevant service personnel, test centre or health authority should be contacted to obtain the test result.  <br/>In order to make it easier for you to find the responsible health authority, the RKI has provided a corresponding tool. By entering the postal code or the city of residence, the responsible health authority (including contact details) is immediately displayed: <a href='https://tools.rki.de/PLZTool/en-GB'>Robert Koch Institute Tool: Health authority by postal code or place</a>",
+                            "Further information on the QR Code procedure can be found in this FAQ entry: <a href='#qr_test'>I have scanned a QR code, but the test result could not be retrieved for days.</a>",
                             "The websites of the Federal Government are offering general information and videos: <a href='https://www.bundesregierung.de/breg-de/themen/corona-warn-app'>https://www.bundesregierung.de/breg-de/themen/corona-warn-app</a>.",
                             "The Robert Koch Institute also provides information on epidemiological aspects: <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/nCoV_node.html'>https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/nCoV_node.html</a>.",

```

Comment:
The link
https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/nCoV_node.html
is German language.

Unfortunately it seems that
https://www.rki.de/EN/Content/infections/epidemiology/outbreaks/COVID-19/COVID19.html
and
https://www.rki.de/EN/Content/infections/epidemiology/outbreaks/COVID-19/CWA/CWA.html
(which is still at the 1.9 level) are not being maintained in the English language any more.

Probably that should be checked with RKI to see if they intend to maintain the English language pages in future or if they have frozen them.

Reply:
I've added a disclaimer that the link is german only for now.

You can make an issue for this and we'll bring this forward to the RKI. Don't know how long this will take since everyone is currently occupied with the 2.0 release.

Reply:
Issue https://github.com/corona-warn-app/cwa-website/issues/1127 opened.